src/Simplex_tree/example/simplex_tree_multi.cpp

```diff
@@ -0,0 +1,56 @@
+/*    This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+ *    See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+ *    Author(s):       David Loiseaux
+ *
+ *    Copyright (C) 2023 Inria
+ *
+ *    Modification(s):
+ *      - YYYY/MM Author: Description of the modification
+ */
+
+#include <gudhi/Simplex_tree.h>
+#include <gudhi/Simplex_tree/Simplex_tree_multi.h>
+
+#include <iostream>
+#include <initializer_list>
+
+struct ST_MULTI {
+public:
```

Comment:
```suggestion
```
Just a detail: the `public` here is useless, as all members of a `struct` are by default public.

Reply:
True, but I have seen style guides that advocate being explicit like this. So if David likes having `public:` I wouldn't force him to remove it.